plugins/modules/apt_rpm.py

```diff
@@ -102,6 +125,20 @@ def update_package_db(module):
     if rc != 0:
         module.fail_json(msg="could not update package db: %s" % err)
 
+    return out
+
+
+def clean(module):
+    rc, out, err = module.run_command([APT_PATH, "clean"], check_rc=True)
+
+    return "Cleaned\n"
```

Comment:
Why is this not returning `out`?

Reply:
Because apt-get clean command does not return any output.